### ISSUE TITLE
Latest update not functional

### ISSUE THREAD

**Issue by webbcreations** [salphonso/wmc2plex.bundle#7]
I am on the latest versions of both wmc2plex and serverwmc and utilize Plex.  I can retrieve the channels, etc. but when I attempt to view a station I receive an error.  Please help.

**Comment by gt2690b** [comment]
+1

edit: using hdgrandslam for my hdhomerun works great

double edit: hdgrandslam only works for DLNA tuners.. hdhr3 doesn't work :(

**Comment by gt2690b** [comment]
Bump

**Comment by nitrouscars** [comment]
I got the same problem.  Can see guide and channels but error when play live tv.  When a fix for the new update? Some people said it is working with an old version of Plex...

**Comment by gt2690b** [comment]
Yes it used to work for me in the old plex.. no more, though :(... says encoder decoder not found

**Comment by TheDaChicken** [comment]
I have this same problem

**Comment by TheDaChicken** [comment]
Try turning off Direct Play and Direct Stream

**Comment by soapdishbandit** [comment]
Doesn't work for me.  Turning off Direct Play/Direct Steam did not work.
